dask/delayed.py

```diff
@@ -334,6 +322,43 @@ def _get_binary_operator(cls, op, inv=False):
     _get_unary_operator = _get_binary_operator
 
 
+class DelayedFunction(Delayed):
```

Comment:
Dask.delayed was heavily used in this PR.  These changes are also present in a separate PR over at #1118